redpanda/templates/NOTES.txt

```diff
@@ -19,16 +19,16 @@ Congratulations on installing {{ .Chart.Name }}!
 
 The pods will rollout in a few seconds. To check the status:
 
-  kubectl -n {{ .Release.Namespace }} rollout status -w statefulset/redpanda
+  kubectl -n {{ .Release.Namespace }} rollout status -w statefulset/{{ .Release.Name }}
```

Comment:
The [statefulSet is named](https://github.com/vectorizedio/helm-charts/pull/18/commits/6136551fbde8b001a3d38b6339b6f43a3c50547d#diff-f74d1889126f524c4b7e7e7836a376c58b0ca5aabe3720e03732b0e25d6ee380L21) `{{ template "redpanda.fullname" . }}`

Reply:
Thanks, fixed!